src/libuv/Native.Settings

```diff
@@ -50,6 +50,8 @@
     <Link>
       <EnableCOMDATFolding>true</EnableCOMDATFolding>
       <OptimizeReferences>true</OptimizeReferences>
+      <ControlFlowGuard>Guard</ControlFlowGuard>
+      <DebugInformationFormat>None</DebugInformationFormat>
```

Comment:
Is it even correct? What is the linker switch this value corresponds to? As per: https://msdn.microsoft.com/en-us/library/ee862477.aspx `None` is not a valid option.